src/common/AndroidUtils.ts

```diff
@@ -236,42 +236,47 @@ export class AndroidSDKUtils {
     > {
         return new Promise<AndroidPackage>(async (resolve, reject) => {
             try {
+                const minSupportedAndroidRuntime: Version = new Version(
+                    androidConfig.minSupportedAndroidRuntime
+                );
                 const packages = await AndroidSDKUtils.fetchInstalledPackages();
                 if (packages.size < 1) {
                     return reject(
                         new Error(
-                            `Could not find android api packages. Requires any one of these [${
-                                androidConfig.supportedRuntimes[0]
-                            } - ${
-                                androidConfig.supportedRuntimes[
-                                    androidConfig.supportedRuntimes.length - 1
-                                ]
-                            }]`
+                            `Could not find any supported Android API packages. Minimum supported Android API package version is ${androidConfig.minSupportedAndroidRuntime}`
```

Comment:
Minor: this message is a bit misleading. Can we change the first sentence to:

> No Android API packages are installed.

Reply:
To clarify, one thing I specifically wanted to strike from this content is the word "supported". This error denotes that *no* API packages were found. I think the message would be more clear/specific if we indicated that nothing was installed. "Supported" sounds like they've got something, but it's not what we need.

Reply:
Oh wait I think you haven't pushed updates yet, sorry. 😉 Well it was probably good to clarify anyway. 🤣